src/css/support.js

```diff
@@ -0,0 +1,53 @@
+import jQuery from "../core.js";
+import document from "../var/document.js";
+import documentElement from "../var/documentElement.js";
+import support from "../var/support.js";
+
+( function() {
+
+var reliableTrDimensionsVal,
+	div = document.createElement( "div" );
+
+// Finish early in limited (non-browser) environments
+if ( !div.style ) {
+	return;
+}
+
+jQuery.extend( support, {
```

Comment:
Is it worth using `jQuery.extend` here? Wouldn't assigning directly to `support` be smaller?

Reply:
Probably worth switching to assignment. Was an artifact from copying over.

Reply:
Switching to assignment saved us 8 bytes.